### DISCUSSION TITLE
Libby App

### DISCUSSION THREAD

**Discussion by EjReece26** [FrostCo/AdvancedProfanityFilter#481] [General]
I’m trying to find a proper config for using the extension for Libby app.com
Any help would be appreciated. Has anyone else gotten it to work?

**Comment by richardfrost** [comment]
What isn't working with it? I don't have any experience with the website yet, but I did test out [this sample](https://libbyapp.com/open/sample/315/4067043) and added the word `drift` to my filter list to test and it appears to be working on the book text. Is there something else that isn't working? Maybe you could share a screenshot?

**Reply by EjReece26** [reply]
I'm running through libbyapp.com and there is no filtering whatsoever of any language. I added the custom config below to see if it would work, but that did not work either. Sorry, I'm very new to this and obviously I'm missing something. 

```json
{
  "libbyapp.com": [
    {
      "mode": "cue",
      "videoCueLanguage": "en",
      "videoSelector": "video.top-media-element"
    }
  ]
}
```

**Reply by richardfrost** [reply]
Are you trying to use it on text (ebook), or an audiobook?

**Reply by EjReece26** [reply]
Ebook, through Libby app.com

**Reply by EjReece26** [reply]
I think I may just be stuck on what boxes I need checked in settings in order to get the words removed from the libbyapp

**Reply by richardfrost** [reply]
@EjReece26 You shouldn't need to do custom config if its just for filtering the text on the page. Can you try visiting the sample from my first comment and see if it works (if you also temporarily add the word "drift" to the filter)? It should pretty much just work out of the box without any special config, at least for that sample book I found. Its possible it could be different for other books, but let's confirm whether it works or not on that book for you.

**Reply by JacobChild** [reply]
@richardfrost I just checked on Libby and mine worked out of the box for an ebook, thanks so much! I would absolutely love audiobook support in libby or on audible as well as an fyi!
